mailpoet/assets/js/src/email-editor/engine/components/block-editor/block-editor.tsx

```diff
@@ -122,82 +125,92 @@ export function BlockEditor() {
   }
 
   return (
-    <BlockEditorProvider
-      value={blocks}
-      onInput={onInput}
-      onChange={onChange}
-      settings={settings}
-      useSubRegistry={false}
+    // BlockContextProvider has set templateSlug because the hook `useGlobalStylesOutputWithConfig` doesn't render layout padding without it.
+    <BlockContextProvider
+      value={{
+        postId,
+        postType: 'mailpoet_email',
+        templateSlug: 'mailpoet-email',
```

Comment:
Just a detail. It looks a bit weird to see `mailpoet_email` and `mailpoet-email`. Could we maybe use `mailpoet_email_template`? If I understand correctly it doesn't matter what we use, right?

Reply:
👍 Yes, it doesn't matter about the name. Changed here: https://github.com/mailpoet/mailpoet/pull/5499/commits/c7ad438e5e703a286a60154eb0d081764459a242